### ISSUE TITLE
Receipt: custom email 

### ISSUE THREAD

**Issue by pdpinch** [mitodl/mitxonline#532]
As a user, I'd like to get a complete and customized receipt via email.

#### Designs and Mockups

As close as possible to what we do for xPRO:

<img width="940" alt="image" src="https://user-images.githubusercontent.com/430126/164752128-3a07dbc4-ab54-43d8-bb0a-fa8dd2344148.png">

#### Acceptance Criteria:

- [ ] same add'l information as #531 

#### Related issues

- #531

**Comment by pdpinch** [comment]
On Hold until @rhysyngsun provides some guidance on how we can put this into an ol-django library.

**Comment by jkachel** [comment]
Question (probably for @rhysyngsun) - do we need the more complete user record for this (like xPro has, which includes address fields and such) or can I pull that data from the CyberSource payload? My thinking is that, since the receipt is a record of a particular transaction, the address should be the one that was supplied in the checkout process (which may be different from the learner's actual address), but xPro does this differently.

**Comment by rhysyngsun** [comment]
@jkachel I agree that it should match the actual payment information from cybersource.
